Dockerfile

```diff
@@ -57,7 +59,8 @@ RUN gradle --console=plain \
     -x :hub-saml:jar \
     -x :hub-saml-test-utils:jar
 
-FROM openjdk:11.0.9.1-jre
+ARG REGISTRY_IMAGE_JDK=openjdk:11.0.9.1-jre
```

Comment:
If the build arg is set in the pipeline (as in your infrastructure-config PR), does it take precedence over the value set here?

Reply:
I checked - yes! https://docs.docker.com/engine/reference/builder/#arg